corehq/apps/hqwebapp/templates/hq-captcha-field.html

```diff
@@ -0,0 +1,18 @@
+{{image}}{{hidden_field}}<br />{{text_field}}
+<script>
+$(function() {
+    // http://stackoverflow.com/a/20371801
+    $('img.captcha').after(
+        $('<span> <button class="captcha-refresh">' +
+          '<i class="fa fa-refresh icon icon-refresh"></i></button></span>')
+    );
+    $('.captcha-refresh').click(function(){
+        var $form = $(this).parent().closest('form');
+        $.getJSON("/captcha/refresh/", {}, function(json) {
+            $form.find('input[name$="captcha_0"]').val(json.key);
+            $form.find('img.captcha').attr('src', json.image_url);
+        });
+        return false;
+    });
+});
+</script>
```

Comment:
Inlined this script to keep things super simple, but could be persuaded to move it to a separate .js file if this seems dirty.